compio-runtime/src/attacher.rs

```diff
@@ -15,32 +21,30 @@ use crate::Runtime;
 /// itself is Send & Sync. We mark it !Send & !Sync to warn users, making them
 /// ensure that they are using it in the correct thread.
 #[derive(Debug, Clone)]
-pub struct Attacher {
+pub struct Attacher<S> {
```

Comment:
`Attacher` is now `Send/Sync` if `S` is. Is this desirable? If yes, please update the comment.

Reply:
Yes.